tdrs-backend/tdpservice/scheduling/parser_task.py

```diff
@@ -20,28 +22,51 @@ def parse(data_file_id, should_send_submission_email=True):
     # passing the data file FileField across redis was rendering non-serializable failures, doing the below lookup
     # to avoid those. I suppose good practice to not store/serializer large file contents in memory when stored in redis
     # for undetermined amount of time.
-    data_file = DataFile.objects.get(id=data_file_id)
+    try:
```

Comment:
I want to confirm the logic here. Is the following steps accurate?

1. file parsing starts and status is set to `Pending` during this process. 
2. parsing processing completes and the status of parsing is retrieved from summary object
3. parsing metadata from the summary object retrieved
4. data submission email sent to approved users associated with the file/STT


if an exception is encountered along the way (i assume during parsing?):
- if its a database exception, the exception is logged (in LogEntries?) and the task exits
- if some other exception, a generic exception-related error message is added to the feedback report, parsing status is set to Rejected,  the exception is logged (in LogEntries?), and the task exits.

Reply:
That is correct, would it make more sense to have both exception blocks perform the same handling? I.e LogEntry creation, reject file, and generate an error? Just for clarity, we generate the error to ensure the STT has the required info to notify an admin of the issue since we don't have a monitoring solution or email based solution in place yet.